lib/suse/connect/api.rb

```diff
@@ -75,6 +77,27 @@ def forward_system_activations(system)
         )
       end
 
+      def send_bulk_system_update(systems, system_limit = nil)
+        # Test system limit
+        system_limit ||= BULK_SYSTEM_REQUEST_LIMIT
+        last_response = nil
+        systems.each_slice(system_limit) do |batched_systems|
+          params = prepare_payload_for_bulk_update(batched_systems)
+          last_response = make_single_request(
+            :put,
+            "#{connect_api}/organizations/systems",
+            { body: params.to_json }
+          )
+        end
+        last_response
```

Comment:
Is there a reason for the method to return the `last_response`?

Reply:
Currently we need to return something for testing. Next, it will be `yield`-ing the results so we can updated the systems on RMT side as well.